### ISSUE TITLE
Model won't train with tensorflow 1.11.0

### ISSUE THREAD

**Issue by moranlouie** [iperov/DeepFaceLab#50]
I am receiving this error message:
"terminate called after throwing an instance of 'std::bad_alloc'"

This is after I upgraded tensorflow to 1.11.0 and Keras to 2.2.4. I upgraded to solve this error: "importerror: cannot import name 'normalize_data_format'"

**Comment by TooMuchFun** [comment]
The `normalize_data_format` issue was in resolved https://github.com/iperov/DeepFaceLab/pull/45. Have you merged in upstream master since then?

**Comment by TooMuchFun** [comment]
Please refer to existing issues regarding tensorflow and CUDA updates. https://github.com/iperov/DeepFaceLab/issues/48. We are waiting for tensorflow 1.13 to ship before migrating. This version is required before we can support updates to CUDA, dlib, and tensorflow-gpu.
